### ISSUE TITLE
add custom validation message to IsDate, isAlpha, Is ,.....

### ISSUE THREAD

**Issue by karimabdelhakim** [RobinBuschmann/sequelize-typescript#261]
for example `@IsDate(msg:"please enter a valid date")`

**Comment by RobinBuschmann** [comment]
@karimabdelhakim yeah, this would be useful and should be implemented. For now you can achieve this like so: 
```ts
@Column({
  ...,
  validate: {
    isDate: {msg: 'your message'}
  }
})
```

Any PRs are welcome :) ;)

**Comment by jrcss11-iscteiulpt** [comment]
Some update about this feature?

**Comment by contra** [comment]
You can see this is implemented for some validators but not all of them: https://github.com/RobinBuschmann/sequelize-typescript/blob/master/src/validation/length.ts

You would basically just need to copy this to every validator in this folder, it wouldn't be a massive lift.
